gamp-recipes/app/components/RecipesCard.tsx

```diff
@@ -14,30 +13,56 @@ type recipeProps = {
 }
 
 
-export default function RecipesCard({ id, image, title, type }: recipeProps): JSX.Element {
+export default function RecipesCard({ id, image, title, category, alcoholic }: recipeProps): JSX.Element {
+	function capitalize(word: string): string {
+		const capitalized = word.split('');
+		capitalized[0] = capitalized[0].toUpperCase();
+
+		return capitalized.join('');
+	}
+	
 	return (
-		<div className={ `text-center mb-4 relative flex flex-col gap-2
+		<div className={ `text-center mb-4 relative flex flex-col gap-2 
 		xl:w-64 xl:mx-auto` }>
-			<Link href={`/${id}`} replace>
-				<picture>
-					<img
-						alt={title}
-						src={image}
-						className='object-cover rounded-3xl w-96 h-48 shadow-lg'
-					/>
-				</picture>
-			</Link>
-			<section className='px-4 flex items-left text-left justify-between'>
-				<div className='flex flex-col gap-2'>
-					<h2 className="text-[1.75rem] font-black">{title}</h2>
-					<p className={ `text-[0.8rem] w-20 text-white text-center rounded-xl uppercase font-[600] tracking-[0.075rem] ${ type === 2 ? 'bg-red' : 'bg-blue-600' }` }>
-						{ type === 2 ? 'meal' : 'drink' }
-					</p>
+			<Link href={`/${id}`}>
+				<div
+					style={{
+						backgroundImage: `url(${image})`,
+						backgroundRepeat: 'no-repeat',
+						backgroundPosition: 'top center',
+						backgroundSize: '100%',
+						width: '336px',
+						height: '256px',
+					}}
+					className='rounded-3xl shadow-lg flex justify-center'
+				>	<section
+						className='grid grid-cols-2 self-center h-[90%]  w-[90%]'
+					>
+						<p
+							className='text-heading3 self-end justify-self-start text-white shadow-lg'
+						>
+							{title}
+						</p>
+						
+						<p
```

Comment:
To pensando aqui se seria uma boa componetizar essa parte, só para o código ficar mais legível.

Reply:
Não acho tão necessário componentizar uma configuração que já é do card, justificaria se fóssemos reutilizar essa config em outros lugares, ter essa configuração em outro arquivo para mim dificultaria o entendimento do card

Uma sugestão é mover o bloco de config pra uma função 
```javascript
	const bgConfig = (image: string) => {
		return {
			backgroundImage: `url(${image})`,
			backgroundRepeat: 'no-repeat',
			backgroundPosition: 'top center',
			backgroundSize: '100%',
			backgroundColor: '#bfbfbf',
			width: '336px',
			height: '256px',
		};
	};
```

e aí chamar no jsx assim melhorando a legibilidade do componente no retorno
```html
	<div
		style={bgConfig(image)}
		className='rounded-3xl shadow-lg flex justify-center z-0 bg-blend-multiply'
	>	
```

Reply:
boa!